Doc/library/io.rst

```diff
@@ -273,7 +273,7 @@ I/O Base Classes
       with open('spam.txt', 'w') as file:
           file.write('Spam and eggs!')
 
-   :class:`IOBase` provides these data attributes and methods:
+   :class:`IOBase` provides these data attribute and methods:
```

Comment:
We should still use "attributes" because we have "these" in front of "data attributes and methods".
```suggestion
   :class:`IOBase` provides these data attributes and methods:
```

Reply:
Alright.